README.md

```diff
@@ -23,9 +23,21 @@ In your Jenkins project, install the plugin.
   3. Click **Choose File** and select the IBM Bluemix DevOps plugin installation file. 
   4. Click **Upload**.
   5. Restart Jenkins and verify that the plugin was installed.
-
+  
 ## 3. Integrate DevOps Insights with your Jenkins project
 
+Configure your Jenkins jobs to send notifications to your toolchain.
+
+Configure the Jenkins tool integration by following the instructions in the [Bluemix Docs](https://console.ng.bluemix.net/docs/services/ContinuousDelivery/toolchains_integrations.html#jenkins).
+
+**Note:** You do not need to follow the instructions to install this plugin.
+
+### Example configurations
+  - Configuring the ICD_WEBHOOK_URL for job configurations: ![Set ICD_WEBHOOK_URL Parameter](https://github.com/imvijay2007/Jenkins-IBM-Bluemix-Toolchains/blob/notification_1/screenshots/Set-Parameterized-Webhook.png "Set Parameterized WebHook")
+  - Configuring post-build actions for job notifications: ![Post-build Actions for WebHook notification](https://github.com/imvijay2007/Jenkins-IBM-Bluemix-Toolchains/blob/notification_1/screenshots/PostBuild-WebHookNotification.png "Configure WebHook Notification in Post-build Actions")
+
+## 4. Integrate DevOps Insights with your Jenkins project
+
 After the plugin is installed, you can integrate DevOps Insights into your Jenkins project. 
 
 To view the dashboard or create a policy for the quality gate that you are going to use for your project, go to the [control center](https://control-center.ng.bluemix.net/).
```

Comment:
the post build action name in the general flow section has to be update to IBM Cloud DevOps